druid-shell/examples/shello.rs

```diff
@@ -64,9 +64,17 @@ impl WinHandler for HelloState {
     }
 
     fn key_down(&mut self, event: KeyEvent) -> bool {
-        let deadline = std::time::Instant::now() + std::time::Duration::from_millis(500);
-        let id = self.handle.request_timer(deadline);
-        println!("keydown: {:?}, timer id = {:?}", event, id);
+        // TODO(x11/timers): Timers not implemented yet on X11, so this won't work there.
+        cfg_if::cfg_if! {
```

Comment:
okay this is some merge mess? I have no idea what happened. 😕

Reply:
Nope, this code currently breaks because we don't have timers implemented in X11 yet, so I `cfg_if`'d it away for X11.

Reply:
what I would do is implement the API, returning a timer token, but also log a warning that timers don't work?

Reply:
Oh right, I *did* just do that :P I'll look into it.

Reply:
Reverted.